e2e/test/scenarios/dashboard-cards/click-behavior.cy.spec.js

```diff
@@ -943,17 +925,24 @@ describe("scenarios > dashboard > dashboard cards > click behavior", () => {
         "have.text",
         "Created At is Jul 1–31, 2022",
       );
-      cy.location().should(({ hash, pathname }) => {
-        expect(pathname).to.equal("/question");
-
-        const card = deserializeCardFromUrl(hash);
-        expect(card.name).to.deep.equal(TARGET_QUESTION.name);
-        expect(card.display).to.deep.equal(TARGET_QUESTION.display);
-        expect(card.dataset_query.query).to.deep.equal({
-          ...TARGET_QUESTION.query,
-          filter: QUERY_FILTER_CREATED_AT,
-        });
-      });
+
+      cy.location("pathname").should("equal", "/question");
```

Comment:
The 3 tests updated in this file were testing implementation details (with `deserializeCardFromUrl`).
With `verifyNotebookQuery` - not anymore!